app/controllers/dashboard_controller.rb

```diff
@@ -2,5 +2,19 @@ class DashboardController < ApplicationController
   def index
     debate = Debate.find_by!(slug: params[:slug])
     @debate = DebatePresenter.new(debate)
+
+    @right = {
+      positive: true,
+      value: @debate.positive_value,
+      percent: @debate.positive_percent,
+      count: @debate.positive_count
+    }
+    
+    @left = {
```

Comment:
Wydaje mi się, że powinniśmy te zmienne nazwać inaczej. `@left` i `@right` nic nie mówią, z tego co rozumiem, to lepszymi nazwami byłyby `@positive` i `@negative`, albo nawet `@positive_data` i `@negative_data`.